bindings/pyroot/cppyy/cppyy/test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 1995-2025, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+add_custom_target(cppyy_tests_dict ALL COMMAND make all WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/bindings/pyroot/cppyy/cppyy/test)
+
+file(GLOB CPPYY_TEST_SRC "test_*.py")
```

Comment:
We try to avoid globs in general: could we perhaps be explicit listing the test files we want to add to the list?